lib/govuk_content_schema_examples.rb

```diff
@@ -21,8 +21,8 @@ module GovukContentSchemaExamples
     include GdsApi::TestHelpers::ContentStore
 
     # Returns a hash representing an finder content item from govuk-content-schemas
-    def govuk_content_schema_example(name)
-      string = GovukContentSchemaTestHelpers::Examples.new.get('finder', name)
+    def govuk_content_schema_example(name, format=nil)
```

Comment:
Rather than default this parameter to `nil` and provide a default value where it's used, it would be better to just add the default value here:

```ruby
def govuk_content_schema_example(name, format="finder")
  string = GovukContentSchemaTestHelpers::Examples.new.get(format, name)
  JSON.parse(string)
end
```